tests/run_tests_py3.sh

```diff
@@ -17,4 +17,4 @@ python3 -m pytest tests/epyccel/test_epyccel_functions.py
 python3 -m pytest tests/epyccel/test_epyccel_modules.py
 python3 -m pytest tests/epyccel/test_arrays.py
 python3 -m pytest tests/epyccel/test_loops.py
-
```

Comment:
@EmilyBourne Can you add this line also to file `.travis.yml` in the root directory?